plugins/window_size/windows/window_size_plugin.cpp

```diff
@@ -245,6 +247,46 @@ void WindowSizePlugin::HandleMethodCall(
     ::ShowWindow(GetRootWindow(registrar_->GetView()),
                  *visible ? SW_SHOW : SW_HIDE);
     result->Success();
+  } else if (method_call.method_name().compare(kEnterFullScreenMethod) == 0) {
+    HWND hwnd = GetRootWindow(registrar_->GetView());
+    HDC hdc = GetDC(hwnd);
+    DEVMODE fullscreenSettings;
+    bool isChangeSuccessful;
+    EnumDisplaySettings(NULL, 0, &fullscreenSettings);
+    fullscreenSettings.dmPelsWidth = GetDeviceCaps(hdc, DESKTOPHORZRES);
+    fullscreenSettings.dmPelsHeight = GetDeviceCaps(hdc, DESKTOPVERTRES);
+    fullscreenSettings.dmBitsPerPel = GetDeviceCaps(hdc, BITSPIXEL);
+    fullscreenSettings.dmDisplayFrequency = GetDeviceCaps(hdc, VREFRESH);
+    fullscreenSettings.dmFields = DM_PELSWIDTH | DM_PELSHEIGHT | DM_BITSPERPEL | DM_DISPLAYFREQUENCY;
+    SetWindowLongPtr(GetRootWindow(registrar_->GetView()), GWL_EXSTYLE, WS_EX_APPWINDOW | WS_EX_TOPMOST);
+    SetWindowLongPtr(GetRootWindow(registrar_->GetView()), GWL_STYLE, WS_POPUP | WS_VISIBLE);
+    SetWindowPos(GetRootWindow(registrar_->GetView()), HWND_TOPMOST, 0, 0, GetDeviceCaps(hdc, DESKTOPHORZRES), GetDeviceCaps(hdc, DESKTOPVERTRES), SWP_SHOWWINDOW);
+    isChangeSuccessful = ChangeDisplaySettings(&fullscreenSettings, CDS_FULLSCREEN) == DISP_CHANGE_SUCCESSFUL;
+    ShowWindow(GetRootWindow(registrar_->GetView()), SW_MAXIMIZE);
+    result->Success(
+      flutter::EncodableValue(isChangeSuccessful)
```

Comment:
Why is failure being returned as a success? And why do the return semantics of Windows and Linux not match (nor does this match the use on the Dart side, which ignores the return value).